app/styles/main.scss

```diff
@@ -1,105 +1,163 @@
 @import "variables";
 
 body {
-  color: $blue;
-  background-size: cover;
-  width: 100%;
+color: $blue;
+background-size: cover;
+width: 100%;
 }
 
 #footer {
-  padding-top: 50px;
-  background-color: $black;
-  a{
-    color:$black;
-  }
-  center{
-    color:$white;
-    background-color: $black;
-  }
+padding-top: 50px;
+background-color: black;
+a:hover{
+color:white;
+}
+center{
+color:white;
+background-color: black;
+}
 }
 .colorfix {
-  background-color: $blue;
+background-color: black;
 }
 
 .content {
-  background-color: $white;
-  color: black;
+background-color: $white;
+color: black;
 }
 
 
 .bs-example {
-  margin: 20px;
+margin: 20px;
 }
 
 .btn btn-danger, btn-primary, a, hover {
-  color: $white;
+color: $white;
 }
 
 .btn-head {
-  font-size: 100%;
+font-size: 100%;
 }
 
 .carousel {
-  margin-top: 20px;
+margin-top: 20px;
 }
 
 h2 {
-  margin: 0;
-  color: #666;
-  padding-top: 90px;
-  font-size: 52px;
-  font-family: "trebuchet ms", sans-serif;
+margin: 0;
+color: #666;
+padding-top: 90px;
+font-size: 52px;
+font-family: "trebuchet ms", sans-serif;
 }
 
 .item {
-  background: #333;
-  text-align: center;
-//  height: 300px !important;
+background: #333;
+text-align: center;
+height: 300px !important;
 }
 
 .pad-head {
-  padding: 30px 10px 0px 10px;
+padding: 30px 10px 0px 10px;
 }
 
 .row-head {
-  background-color: $white;
-  height: 200px;
+background-color: $white;
+height: 200px;
 }
 
 .main_title {
-  // background-color: white;
-  // color: green;
-  -webkit-box-shadow: inset 0 0 100px rgba(0,0,0,.5);
-
-  box-shadow: inset 0 0 100px rgba(0,0,0,.5);
-
-  h1{
-    // color: black;
-  }
-
-  .btn-danger {
-    width: auto;
-    height: auto;
-  }
+background-color: white;
+color: black;
+-webkit-box-shadow: inset 0 0 100px rgba(0,0,0,.5);
 
-  text-align: center;
+box-shadow: inset 0 0 100px rgba(0,0,0,.5);
 
+h1{
+color: black;
 }
 
-// .navbar-default navbar-collapse in {
-//   border-color: blue;
-// }
-.landing-title{
-  color: $grey;
+.btn-danger {
+width:auto;
```

Comment:
Properties should be ordered height, width<br>Colon after property should be followed by one space